### PR TITLE
JESI-2977

### PR DESCRIPTION
**Pull request by BadAlgorithm**
[JESI-2977](https://jesims.atlassian.net/browse/JESI-2977)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 3.1.1
```
Comment (axrs) [inline CHANGELOG.md]: 3.2.0 We added mod to backpack.

Breaking.Added.Fixed

File: VERSION [hunk 2]
```diff
@@ -1 +1 @@
-3.1.0
```
Comment (axrs) [inline VERSION]: 3.2.0

File: test/io/jesi/backpack/number_test.cljc [hunk 3]
```diff
@@ -25,3 +25,17 @@
...
+
+    (testing "when non 0"
+      (is= 4M (bp/mod 10 6))
+      (is= 6M (bp/mod 6 10))
+      (is= 0.1M (bp/mod 5.3 1.3)))
+
+    (testing "when 0"
+      (is= 0M (bp/mod 1.0 1.0))
+      (is= 0M (bp/mod 18 1.5))
+      (is= 0M (bp/mod 5.2 1.3)))))
```
Comment (axrs) [inline test/io/jesi/backpack/number_test.cljc]: add a test for `(is (zero? (mod 0.3 0.1))` and `(is (= 0.1 (mod 0.1 0.3)))`
